fastlane/metadata/android/de-DE/short_description.txt

```diff
@@ -0,0 +1 @@
+File synchronization client
```

Comment:
This is still in English. "Client zum synchronisieren von Daten"?

Reply:
@mueller-ma you are the PR owner. Please, could you change it? After this, if you could rebase the branch against master, we will merge it.